kcidev/libs/maestro_common.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import json
+
+import click
+
+from kcidev.libs.common import *
+
+
+def maestro_print_api_call(host, data=None):
+    click.secho("maestro api endpoint: " + host, fg="green")
```

Comment:
this will print to stdout, which IMO should not be the case. This is useful information to have as part of logging and should go to stderr, this way we can keep stdout clean for outputting json data (or any parseable format) that can then be piped into `jq` or such to extract meaningful data.
maybe have a `kci_info` helper that prints in green to stderr?

Reply:
that's a good point